release/models/interfaces/openconfig-if-aggregate.yang

```diff
@@ -204,12 +213,61 @@ module openconfig-if-aggregate {
       type leafref {
         path "/oc-if:interfaces/oc-if:interface/oc-if:name";
       }
+      status deprecated;
       description
         "Specify the logical aggregate interface to which
         this interface belongs";
     }
   }
 
+  grouping aggregation-ethernet-config {
+    description
+      "Adds configuration items for dedicated aggregation container
+      for Ethernet interfaces belonging to a logical aggregate / LAG";
+
+    leaf aggregate-id {
```

Comment:
We actually explicitly made this not part of a Ethernet configuration to support the idea of bundles of non-Ethernet interfaces.

(Specifically, networks that I've worked on have used POS bundles, and this is relatively widely supported -- `bundle-POS` on XR, `as` on JUNOS. Of course, good old POS isn't that popular these days, and no-one has yet extended OpenConfig to support SONET/SDH interfaces at all AFAIK. This being said, it still would be possible, and given we've already made the choice to accommodate it, I'm not clear we should change it now.)

Reply:
If we use a  `/lacp/interfaces/interface` approach then we are decoupled with the L1/L2 transport.

Reply:
@dplore would you mind expanding a bit on the `/lacp/interfaces/interface` approach? As detailed in my other comment, I'm not sure how individual Ethernet interface `port-priority` fits at `/lacp/interfaces/interface/config`, since that container configures aggregate interfaces. I need a way to configure the priority of individual Ethernet interfaces in a LAG, that's why I initially proposed to add a config container under `/lacp/interfaces/interface/members/member`. But I'm happy to try out any other approach

Reply:
I agree with the approach to use the members container.  Using the approach at https://github.com/openconfig/public/pull/943#discussion_r1302002147 will move (deprecate) the aggregate-id out of the Ethernet configuration and resolve this comment.

Reply:
Would just want to make sure to preserve the `aggregate-id` for the state node since it would be beneficial to enumerate interfaces to figure out which parent they are a member of.

Reply:
Happy to keep `/interfaces/interface/ethernet/state/aggregate-id`, but one thing I wonder is if I deprecate `/interfaces/interface/ethernet/config/aggregate-id`, how else would one configure a physical interface to be part of a static/non-LACP aggregate? Doesn't seem to me like `/lacp/interfaces/interface/members/member/config` would be the right place for that

Reply:
After re-reading, I agree, we shouldn't deprecate `/interfaces/interface/ethernet/config/aggregate-id` after all.  As you state, it is needed to configure ethernet ports into a LAG independent of whether or not LACP is configured.